examples/flutter_gallery/lib/demo/progress_indicator_demo.dart

```diff
@@ -73,6 +74,7 @@ class _ProgressIndicatorDemoState extends State<ProgressIndicatorDemo> with Sing
       new Row(
         mainAxisAlignment: MainAxisAlignment.spaceEvenly,
         children: <Widget>[
+          new CupertinoActivityIndicator(),
```

Comment:
Rather than putting these in the same demos as the material widgets, I think we should create a separate heading in the home page of the gallery and show the widgets separately.